src/app/new-auth/components/input-field/input-field.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, Input, OnInit, Output, EventEmitter, forwardRef } from '@angular/core';
+import { ControlValueAccessor, FormControl, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { AuthValidationService } from 'app/core/auth-validation.service';
+
+@Component({
```

Comment:
Could we add a block comment here to describe how to use this?

Reply:
agreed